File: AzureCodePractice/JavaScript/translate.js
var transEndpoint = 'https://api.cognitive.microsofttranslator.com/';
var transKey = 'a13d0676ecd74de8bd5cc3ebb06d9330';
var transRegion = 'westus2';
var region = 'westus2';


function translate(query, to) {
    var params = `to=${to}`;
    var transUrl = `${transEndpoint}/translate?api-version=3.0&${params}`;

    var data = [{ 'text': query }];
    await $.ajax({
        type: 'POST',
        url: transUrl,
        contentType: 'application/json',
        data: JSON.stringify(data),
        beforeSend:function(xhr){
            xhr.setRequestHeader('Ocp-Apim-Subscription-Key', transKey);
            xhr.setRequestHeader('Ocp-Apim-Subscription-Region', region);
        }
    }).done(function (transResponse) {
        alert(JSON.stringify(transResponse));
        transText = transResponse[0]['translations'][0];
    }).fail(function(err){
        alert(err.responseText)
    })
    return transText
}

File: AzureCodePractice/Python/module/translate.py
import requests

transEndpoint = 'https://api.cognitive.microsofttranslator.com/'
transKey = 'a13d0676ecd74de8bd5cc3ebb06d9330'
transRegion = 'westus2'

def translateText(query, to):    
    params = f'to={to}'
    transUrl = f'{transEndpoint}/translate?api-version=3.0&{params}'
    transHeaders = {
        'Ocp-Apim-Subscription-Key':transKey,
        'Ocp-Apim-Subscription-Region':transRegion,
        'Content-type':'application/json'
    }
    data=[{'text':query}]
    translateResponse = requests.post(transUrl, headers=transHeaders, json=data)

    translatedText = translateResponse[0]['translations'][0]['text']
    return translatedText
